certs/ca_chain/create_certificates.sh

```diff
@@ -0,0 +1,12 @@
+openssl req -x509 -new -nodes -newkey rsa:2048 -keyout fledgeDNP3ca_key.pem -sha256 -days 3600 -out fledgeDNP3ca_cert.pem -subj "/C=US/ST=OR/L=Bend/O=Acme Certificate Corp"
```

Comment:
What is Acme Corp?

Reply:
It's a placeholder for organisation name.

Why might put anything else